indico/modules/events/abstracts/controllers/reviewing.py

```diff
@@ -77,10 +77,13 @@ def _checkParams(self, params):
     def _process(self):
         query = (Abstract
                  .query.with_parent(self.event_new)
+                 .filter(Abstract.state.notin_({AbstractState.duplicate, AbstractState.merged}))
```

Comment:
Makes more sense for sure, just wondering if users realize that they should do it like this